src/values/attachment-item-value.js

```diff
@@ -1,6 +1,21 @@
 import MediaItemValue from './media-item-value';
 
 export default class AttachmentItemValue extends MediaItemValue {
+  constructor(mediaValue, attributes) {
+    super(mediaValue, attributes);
+
+    this.name = attributes.name;
```

Comment:
Is this the filename?

Reply:
Yes. I _think_ we're moving to using `name` instead of `filename` so I went with name here.

Reply:
I'm seeing this in some of the React components: `attachment.filename`
But you're right it says `name` in the attachments service.  I'm going to pull this convo into Slack